### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by Singman33** [datto/dattobd#130]
```
# uname -a
Linux rig1 4.13.0-32-generic #35~16.04.1-Ubuntu SMP Thu Jan 25 10:13:43 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

# more /var/lib/dkms/dattobd/0.9.16/build/make.log
DKMS make.log for dattobd-0.9.16 for kernel 4.13.0-32-generic (x86_64)
dimanche 11 février 2018, 09:48:14 (UTC+0100)
make : on entre dans le répertoire « /usr/src/linux-headers-4.13.0-32-generic »
  CC [M]  /var/lib/dkms/dattobd/0.9.16/build/dattobd.o
/var/lib/dkms/dattobd/0.9.16/build/dattobd.c: In function ‘dattobd_bio_endio’:
/var/lib/dkms/dattobd/0.9.16/build/dattobd.c:330:5: error: ‘struct bio’ has no member named ‘bi_error’
  bio->bi_error = err;
     ^
/var/lib/dkms/dattobd/0.9.16/build/dattobd.c: In function ‘on_bio_read_complete’:
/var/lib/dkms/dattobd/0.9.16/build/dattobd.c:2800:33: error: ‘struct bio’ has no member named ‘bi_error’
  __on_bio_read_complete(bio, bio->bi_error);
                                 ^
scripts/Makefile.build:315 : la recette pour la cible « /var/lib/dkms/dattobd/0.9.16/build/dattobd.o » a échouée
make[1]: *** [/var/lib/dkms/dattobd/0.9.16/build/dattobd.o] Erreur 1
Makefile:1550 : la recette pour la cible « _module_/var/lib/dkms/dattobd/0.9.16/build » a échouée
make: *** [_module_/var/lib/dkms/dattobd/0.9.16/build] Erreur 2
make : on quitte le répertoire « /usr/src/linux-headers-4.13.0-32-generic »
```

**Comment by crawfxrd** [comment]
This was fixed in 08f6f3e. Use 0.10.1 or newer.

**Comment by Singman33** [comment]
Error is yours. Update your documentation when you upgrade the repo.
